src/main/java/plugily/projects/buildbattle/handlers/menu/registry/banner/BannerMenu.java

```diff
@@ -62,74 +63,81 @@ public BannerMenu(Player player, Banner banner) {
     prepareLayerColorStageGui();
   }
 
-  public static void init(Main plugin) {
-    BannerMenu.plugin = plugin;
-  }
-
   @SuppressWarnings("deprecation")
   private void prepareBaseStageGui() {
     NormalFastInv gui = new NormalFastInv(6, new MessageBuilder("MENU_OPTION_CONTENT_BANNER_INVENTORY_COLOR").asKey().build());
+
     for(DyeColor color : DyeColor.values()) {
       ItemStack item;
+
       if(ServerVersion.Version.isCurrentEqualOrLower(ServerVersion.Version.v1_12_R1)) {
         item = XMaterial.WHITE_BANNER.parseItem();
         BannerMeta meta = (BannerMeta) item.getItemMeta();
-        if(ServerVersion.Version.isCurrentEqualOrLower(ServerVersion.Version.v1_12_R1)) {
-          meta.setBaseColor(color);
-        } else {
-          ((org.bukkit.block.Banner) item).setBaseColor(color);
-        }
+
+        meta.setBaseColor(color);
```

Comment:
higher 1.12 works?

Reply:
There is a condition above this `item = XMaterial.WHITE_BANNER.parseItem();`